openrouteservice-api-tests/pom.xml

```
@@ -11,29 +11,25 @@
     <url>http://maven.apache.org</url>
 
     <properties>
+        <java.version>11</java.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

use java.version properties instead to hard code the version again.
```suggestion
        <maven.compiler.source>${java.version}</maven.compiler.source>
        <maven.compiler.target>${java.version}</maven.compiler.target>
```